sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2_and_later/async_samples/sample_send_request_async.py

```diff
@@ -43,7 +43,82 @@ async def sample_send_request():
     async with client:
         response = await client.send_request(request)
     response.raise_for_status()
+    response_body = response.json()
+    print(
+        f"Our resource has {response_body['customDocumentModels']['count']} custom models, "
+        f"and we can have at most {response_body['customDocumentModels']['limit']} custom models."
+        f"The quota limit for custom neural document models is {response_body['customNeuralDocumentModelBuilds']['quota']} and the resource has"
+        f"used {response_body['customNeuralDocumentModelBuilds']['used']}. The resource quota will reset on {response_body['customNeuralDocumentModelBuilds']['quotaResetDateTime']}"
+    )
+    
+    # pass with full url and override the api version
+    request = HttpRequest(method="GET", url=f"{client._endpoint}/formrecognizer/info?api-version=2022-08-31")
+    async with client:
+        response = client.send_request(request)
+    response.raise_for_status()
+    response_body = response.json()
+    print(
+        f"Our resource has {response_body['customDocumentModels']['count']} custom models, "
+        f"and we can have at most {response_body['customDocumentModels']['limit']} custom models."
+        f"The quota limit for custom neural document models is {response_body['customNeuralDocumentModelBuilds']['quota']} and the resource has"
+        f"used {response_body['customNeuralDocumentModelBuilds']['used']}. The resource quota will reset on {response_body['customNeuralDocumentModelBuilds']['quotaResetDateTime']}"
+    )
+    
+    # override the api version to v2.1
+    request = HttpRequest(method="GET", url="v2.1/custom/models?op=summary&api-version=v2.1")
+    async with client:
+        response = client.send_request(request)
+    response.raise_for_status()
+    response_body = response.json()
+    print(f"Our account has {response_body['summary']['count']} custom models, "
+          f"and we can have at most {response_body['summary']['limit']} custom models.")
     
+    # pass with full url and override the api version to v2.1
+    request = HttpRequest(method="GET", url=f"{client._endpoint}/formrecognizer/v2.1/custom/models?op=summary&api-version=v2.1")
+    async with client:
+        response = client.send_request(request)
+    response.raise_for_status()
+    response_body = response.json()
+    print(f"Our account has {response_body['summary']['count']} custom models, "
+          f"and we can have at most {response_body['summary']['limit']} custom models.")
+
+
+async def sample_send_request_v2():
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+
+    client = FormTrainingClient(
```

Comment:
It might be worth mentioning in a comment that the default `FormTrainingClient` API version is `v2.1`.

Reply:
Yup, will update.